app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -490,18 +488,6 @@ static void compareSecondaryIndices(String tableName, List<?> indices1, List<?>
                             " is changing the projection.");
                 }
 
-                if (!globalIndex1.getProvisionedThroughput().getReadCapacityUnits().equals(
-                        globalIndex2.getProvisionedThroughput().getReadCapacityUnits())) {
-                    throw new BridgeInitializationException("Table " + tableName + " global index " + indexName +
-                            " is changing the provisioned read capacity.");
-                }
```

Comment:
This prevention might be necessary.  Without it we are running the risk of wiping out data by accident.  The schema check in general enforces data migration for schema changes.